core/src/main/scala/io/qbeast/core/model/QbeastSnapshot.scala

```diff
@@ -0,0 +1,56 @@
+package io.qbeast.core.model
+
+import io.qbeast.IISeq
+
+trait QbeastSnapshot {
+
+  /**
+   * The current state of the snapshot.
+   * @return
+   */
+  def isInitial: Boolean
+
+  /**
+   * Load methods
```

Comment:
Again scaladoc comments should not be used for naming the parts of the source file.